pcsx2/Gif_Unit.h

```diff
@@ -166,8 +166,18 @@ struct Gif_Path {
 	GIF_PATH_STATE state; // Path State
 	Gif_Path_MTVU  mtvu;  // Must be last for saved states
 
-	Gif_Path()  { Reset(); }
-	~Gif_Path() { _aligned_free(buffer); }
+	Gif_Path() {
+		buffer = nullptr;
+		buffSize = 0;
+		buffLimit = 0;
+		dmaRewind = 0;
+		idx = GIF_PATH_Unknown;
+		Reset();
+	}
```

Comment:
Do you really need all those init. As far as I know, the object is only used in Gif_Unit object which call the `Init` function in the constuctor. Maybe it could be better to merge the `constructor` and the `Init` function.